gear/identifier_lookup/src/python/identifier_app/run.py

```diff
@@ -119,6 +119,12 @@ def run(self, context: GearToolkitContext):
         if not identifiers:
             raise GearExecutionError('Unable to load center participant IDs')
 
+        module_name = self.__file_input.get_module_name_from_file_suffix()
+        if not module_name:
+            raise GearExecutionError(
+                'Expect module suffix to input file name: '
+                f'{self.__file_input.filename}')
+
         filename = f"{self.__file_input.filename}-identifier"
```

Comment:
Remove the file extension from input filename? otherwise output filename will be xxxx.csv-identifier.csv